src/main/java/nl/tudelft/pixelperfect/event/parameter/EventParameterCollection.java

```diff
@@ -75,6 +76,24 @@ public String toString() {
   }
 
   /**
+   * Generate a debug string representation for the parameters.
+   * 
+   * @return the debug string.
+   */
+  public String toDebugString() {
+    StringBuilder sb = new StringBuilder();
+    if (map.isEmpty()) {
+      sb.append("None");
+    } else {
+      for (EventParameter param : map.values()) {
+        sb.append(param.getNumberValue());
```

Comment:
No, that's incorrect: we will be using `EventParameterValues` enumeration values most of the time. Think about `EventParameterValues.HOSTILE_SHIP_ARMOR_ENERGY_SHIELD`, which is already used and not rendered properly currently in your debug console. All these enum values have toString methods, which should be used to render their values. If you are rendering their numeric values you only get useless 0's.